setup/src/Magento/Setup/Controller/Navigation.php

```diff
@@ -47,7 +49,7 @@ public function __construct(NavModel $navigation, Status $status, ObjectManagerP
         $this->navigation = $navigation;
         $this->status = $status;
         $this->objectManagerProvider = $objectManagerProvider;
-        $this->view = new ViewModel;
+        $this->view = new ViewModel();
```

Comment:
Why are you instantiating classes with `new` instead of using `Factory` or just Dependency Injection?

Reply:
Fixed